components/GuidedMeditation/GuidedMeditation.js

```diff
@@ -0,0 +1,54 @@
+import Link from "next/link";
+import styled from "styled-components";
+import AudioPlayer from "react-h5-audio-player";
+import "react-h5-audio-player/lib/styles.css";
+
+export default function GuidedMeditation() {
+  return (
+    <>
+      <div>
+        <h3>Meditation: Living Presence – with Body Scan (12:06 min.)</h3>
```

Comment:
It would be nice if the values (i.e. length) would not be hard coded. Maybe you can refactor that in another user story

Reply:
Thanks! For this use-case it would be a lot of work to analyze the audio file which is on an external server and to estimate the length.